astropy/stats/funcs.py

```diff
@@ -790,3 +793,82 @@ def mad_std(data):
 
     # NOTE: 1. / scipy.stats.norm.ppf(0.75) = 1.482602218505602
     return median_absolute_deviation(data) * 1.482602218505602
+
+
+# TODO Note scipy dependency
+def sigma_to_logp(sigma, two_sided=True):
+    """Convert number-of-sigma to log-probability
+
+    Detection probabilities are often naturally expressed as a
+    false positive probability. On the other hand, astronomers
+    are used to thinking of small probabilities in terms of
+    how many standard deviations out on a normal distribution
+    they are. This function converts between these two
+    representations in a way that works even for probabilities
+    too small to represent in floating point (this happens at
+    about 40-sigma for double precision).
+
+    Parameters
+    ----------
+    sigma : array-like
+        The number of sigma
+    two_sided : boolean
```

Comment:
can you add what happens if two_sided is false. I assume its one sided, but we should be clear. same for the inverse function, too.